### DISCUSSION TITLE
Failed to upgrade from 2.3.91 to 2.3.100

### DISCUSSION THREAD

**Discussion by Mooshcakes** [Security-Onion-Solutions/securityonion#7176] [Unsupported Versions]
Soup failed to upgrade, which resulted in so-dockerregistry [missing]. Upgrading from 2.3.91 to 2.3.100
Seems very similar to issue #5655 

Appreciate everything you guys are doing. I am available to share any additional information - and emotionally ready to rebuild the stack if it comes to it. Thanks!

### The soup.log error:
![image](https://user-images.githubusercontent.com/55214481/153483054-9aaa2e0d-564c-49f3-b847-b3c4068372c3.png)

### `so-status` :
![image](https://user-images.githubusercontent.com/55214481/153483316-c966a15a-aed4-4d5b-8bc0-6e78eef6056b.png)

### Running soup again (after failure):
![image](https://user-images.githubusercontent.com/55214481/153483566-bba2e712-ff11-4ffd-940d-75e8bdd72019.png)

### Attempting ` so-docker-refresh`
![image](https://user-images.githubusercontent.com/55214481/153484243-415c3471-0fb0-4dbc-9c4c-b73c221033de.png)

### results of `salt-call state.highstate`
![image](https://user-images.githubusercontent.com/55214481/153484814-a701fc14-ce68-4c13-9ba2-06cceef867d5.png)

**Comment by dougburks** [comment]
Looking at https://github.com/Security-Onion-Solutions/securityonion/discussions/5655, it seems others experienced at least 2 distinct issues:
1. duplicate sid in thresholding pillar
2. third party service using dockerregistry port 5000

Have you checked your deployment for these issues?

If you're not experiencing either of these issues, then our engineers might need to see your complete soup.log if you're looking for any additional advice.

**Comment by m0duspwnens** [comment]
@Mooshcakes Are you able to post a redacted version of your `soup.log`?

Thanks.
